ray-curator/ray_curator/stages/text/classifiers/prompt_task_complexity.py

```diff
@@ -257,7 +257,9 @@ def setup(self, _: WorkerMetadata | None = None) -> None:
         self.model = CustomDeberta.from_pretrained(self.model_identifier, local_files_only=True).cuda().eval()
         self.model.set_autocast(self.autocast)
 
-    def process_model_output(self, outputs: torch.Tensor) -> dict[str, np.ndarray]:
+    def process_model_output(
+        self, outputs: torch.Tensor, _: dict[str, torch.Tensor] | None = None
+    ) -> dict[str, np.ndarray]:
```

Comment:
Incorrect type string 
```suggestion
    ) -> torch.Tensor:
```

Reply:
good catch, realised that the base had incorrect type too